core/src/index/knowhere/knowhere/index/vector_index/IndexType.cpp

```diff
@@ -70,6 +70,8 @@ const char* INDEX_SPTAG_KDT_RNT = "SPTAG_KDT_RNT";
 const char* INDEX_SPTAG_BKT_RNT = "SPTAG_BKT_RNT";
 const char* INDEX_HNSW = "HNSW";
 const char* INDEX_ANNOY = "ANNOY";
+const char* GSI_FLAT_TANIMOTO = "GSI_FLAT_TANIMOTO";
+const char* GSI_FLAT_HAMMING = "GSI_FLAT_HAMMING";
```

Comment:
It seems that the new index types are never used.

If a new index type is defined, the external call is like this.
```
Insert 
CreateIndex
Query
```
If you do not define an index type, the external call is like this.
```
Insert
Query
```
The difference between them is whether there is a lot of reorganization in the data format.
And that is CreateIndex.

Do you need these new index types?